php/Controllers/SettingsController.php

```diff
@@ -14,6 +14,11 @@
 use Coyote\PluginConfiguration;
 use Coyote\WordPressCoyoteApiClient;
 use Coyote\WordPressPlugin;
+use Twig\Loader\FilesystemLoader;
+use Twig\Environment;
+use Twig\TwigFunction;
+
+require(COYOTE_PLUGIN_PATH . 'vendor/autoload.php');
```

Comment:
Is this include necessary? That should already have been handled by the main plugin file.

Reply:
This is indeed unnecessary. I see that the autoload is indeed already required in the main plugin file.